pkg/config/codeready.go

```diff
@@ -11,6 +14,16 @@ type CodeReady struct {
 func NewCodeReady(config ProductConfig) *CodeReady {
 	return &CodeReady{Config: config}
 }
+func (c *CodeReady) GetWatchableCRDs() []runtime.Object {
+	return []runtime.Object{
+		&chev1.CheCluster{
+			TypeMeta: v1.TypeMeta{
```

Comment:
Is it necessary to specify these `TypeMeta` manually in all of these GetWatchableCRDs cases? It usually isn't, but I know there are some cases where it's not but I can't remember what those cases are.

Reply:
yeah it is, so the detector can pick up the values to compare against the API results.